pylint/lint/pylinter.py

```diff
@@ -460,10 +460,8 @@ def enable_fail_on_messages(self) -> None:
                         # message starts with a category value, flag (but do not enable) it
                         self.fail_on_symbols.append(msg.symbol)
 
-    def any_fail_on_issues(self):
-        return self.stats and any(
-            x in self.fail_on_symbols for x in self.stats.by_msg.keys()
-        )
+    def any_fail_on_issues(self) -> bool:
+        return any(x in self.fail_on_symbols for x in self.stats.by_msg.keys())
```

Comment:
Is self.stats always initialized now (even for lib maintainer) ?

Reply:
Yes 😄 

Finally some benefits from may refactors 😄